src/components/breadcrumbs/breadcrumbs.tsx

```diff
@@ -0,0 +1,216 @@
+     */
+    @Prop()
+    public divider: string = '›';
+
+    /**
+     * Fired when clicking on buttons (not links!)
+     * inside the breadcrumbs.
+     */
+    @Event()
+    public select: EventEmitter<BreadcrumbItem>;
+
+    @Element()
+    private host: HTMLLimelBreadcrumbsElement;
+
+    private button: HTMLButtonElement;
+    private anchor: HTMLAnchorElement;
+
+    public render() {
+        return (
+            <ol
+                role="navigation"
+                aria-label="Breadcrumb"
+                style={{ '--limel-breadcrumbs-divider': `'${this.divider}'` }}
+            >
+                {this.renderSteps(this.items)}
+                {this.renderLastStep(this.items)}
+            </ol>
+        );
+    }
+
+    public componentWillRender() {
+        // Avoid polluting the "clickable registry" with unreferenced nodes
+        this.removeEnterClickable();
+    }
+
+    public componentDidRender() {
+        this.button = this.host.shadowRoot.querySelector('button');
+        this.anchor = this.host.shadowRoot.querySelector('a');
+        if (this.button) {
+            makeEnterClickable(this.button);
+        }
+
+        if (this.anchor) {
+            makeEnterClickable(this.anchor);
+        }
+    }
+
+    public disconnectedCallback() {
+        this.removeEnterClickable();
+    }
+
+    private renderSteps = (items: BreadcrumbItem[]) => {
+        const allStepsWithoutLast = items.slice(0, -1);
+
+        if (this.areItemsLinks(items)) {
+            return allStepsWithoutLast.map(this.renderAsLink);
+        }
+
+        return allStepsWithoutLast.map(this.renderAsButton);
+    };
+
+    private renderAsButton = (item: BreadcrumbItem) => {
+        return [
+            <button
+                role="listitem"
+                id={createRandomString()}
+                class="step"
+                onClick={this.handleClick(item)}
+            >
+                {this.renderIcon(item)}
+                {this.renderLabel(item)}
+            </button>,
+            this.renderTooltip(item),
+        ];
+    };
+
+    private renderAsLink = (item: BreadcrumbItem) => {
+        return [
+            <a
+                role="listitem"
+                id={createRandomString()}
+                class="step"
+                href={item.link.href}
+                title={item.link.title}
+            >
+                {this.renderIcon(item)}
+                {this.renderLabel(item)}
+            </a>,
+            this.renderTooltip(item),
+        ];
+    };
+
+    private renderLastStep = (items: BreadcrumbItem[]) => {
+        const lastItem = items.slice(-1);
+
+        return (
+            <li
+                class="last step"
+                tabindex="-1"
+                aria-current={this.areItemsLinks(items) ? 'page' : 'step'}
+            >
+                {this.renderIcon(lastItem[0])}
+                <span class="text">{lastItem[0].text}</span>
+            </li>
+        );
+    };
```

Comment:
The reason we did it was [this guideline from nngroup](https://www.nngroup.com/articles/breadcrumbs/), which states:
> **5. In the breadcrumb trail, the breadcrumb corresponding to the current page should not be a link.**
You should never have a link that does nothing. The last breadcrumb (denoting the current page) should not be a link. To avoid confusing users, visually differentiate between the current page and the preceding linked breadcrumb items, preferably using underlined or blue text.


If we _only_ had buttons in the structure, we could probably just make the last `button` `disabled` and give it give it a `tabindex="-1"`. But the component can also have links. The HTML <a> (anchor) element itself doesn't have a built-in `disabled` attribute like some other form elements such as `<input>` or `<button>`. It's also not enough to rely on using CSS hacks (such as adding `pointe-events: none;`) to make the link non-clickable. Because users would still be able to tab into it and press <kbd>Enter</kbd> to activate the link. 

Therefore, we decided to render the last item as a `li` that holds a `span`.